capella_rm_bridge/__main__.py

```diff
@@ -147,7 +147,7 @@ def main(
 
     config = yaml.safe_load(conffile)
     params = config["model"]
-    if "git" in config["model"]["path"]:
+    if config["model"]["path"].startswith("git+"):
```

Comment:
This does not detect git URLs that start with `git://`, nor the scp-style short form.

CapellaMBSE provides a helper function `capellambse.split_protocol` which you can use to reliably detect all three variants.